web-app/src/screens/Console/Tenants/TenantDetails/pods/PodDescribe.tsx

```diff
@@ -305,13 +305,15 @@ const PodDescribeTable = ({
               </TableRow>
             </TableHead>
             <TableBody>
-              {items.map((item, i) => (
-                <TableRow key={i}>
-                  {columns.map((column, j) => (
-                    <TableCell key={j}>{item[column]}</TableCell>
-                  ))}
-                </TableRow>
-              ))}
+              {items
+                .filter((item) => item !== null)
```

Comment:
alternatives cc @bexsoft @kaankabalak ?

Reply:
We can do two things here, the first one is to store only valid values during the API response, this way whenever we need these values, we will always have the correct data.

The second option would be to remove the filter & do the comparison in the map. in case the value is null, then you can render a `<Fragment />`